### PR TITLE
WIP: Fxa integration tests

### PR DESCRIPTION
**Pull request by jrbenny35**
Solves about 8 of the 11 items on issue #378. Some of those will need to wait until this bug https://bugzilla.mozilla.org/show_bug.cgi?id=1349861 is fixed.

Also, this looks like a lot of changed but it is based on Matt Miller's (linuxwolf) ```fxa``` branch. So once that is merged, it looks a lot less crazy.

### CODE REVIEW

File: requirements/tests.txt [hunk 1]
```diff
@@ -1,5 +1,6 @@
+fxapom==1.10.1
 PyPOM==1.2.0
-pytest==3.2.2
+pytest==3.3.0
 pytest-selenium==1.11.0
 pytest-xdist==1.18.2
-selenium==3.6.0
+selenium==3.8.0
```
Comment (m8ttyB) [inline requirements/tests.txt]: Wow, `3.8.0` already \o/

File: test/integration/tests/test_lockbox.py [hunk 2]
```diff
@@ -1,22 +1,45 @@
...
+    home_page.sign_in(fxa.email, fxa.password)
+    assert home_page.sign_in_button_is_displayed() is False
+
+
+def test_door_hanger_interaction(home_page, selenium, base_url):
+    """Add an entry and test it shows in the door hanger."""
+    home_page.add_entry()
+    lists = home_page.door_hanger.find_entrys()
+    assert '(No Entry Name)' in lists[0].title
+    entry = lists[0].click()
```
Comment (m8ttyB) [inline test/integration/tests/test_lockbox.py]: I understand now what you were saying +1. Nice job on the FxA flows.

**Comment by jrbenny35** [comment]
I'll have to make sure I go through and update the dependencies.

**Comment by devinreams** [comment]
>Does it make sense to merge these commits into #362?

In other words, he'd re-open a PR against your [personal repo/branch](https://github.com/linuxwolf/lockbox-extension/tree/fxa) versus `mozilla-lockbox/master` here? Hmm.. it would definitely make it cleaner and clearer to have this merged into the corresponding feature and have it all passing and go at once... 🤔 

What do you think @jrbenny35?

**Comment by linuxwolf** [comment]
I think that's the right approach @devinreams.  From the "Create Pull Request" on Github, select "compare across forks", and select:
* head fork: `linuxwolf/lockbox-extension`
* branch: `fxa`

**Comment by jrbenny35** [comment]
I think that will work. I was considering that but wasn't sure.

**Comment by linuxwolf** [comment]
I see the the PR against my `fxa` branch.  I think we can close this PR and carry on from there.

**Comment by devinreams** [comment]
Thanks again @jrbenny35. Moved to https://github.com/linuxwolf/lockbox-extension/pull/3

**Pull request closed without merging**